_profiles/stephen_swatman.md

```diff
@@ -0,0 +1,33 @@
+---
+# Required:
+title: Stephen Nicholas Swatman
+country: FR
+
+# Optional (please add github or gravatar for your picture)
+github: stephenswat
+gravatar: null
+homepage: http://v25.nl/
```

Comment:
hmm, I get an E 503 for this

Reply:
My mistake, was a bad configuration. I've had that site under "temporary" maintenance for a while now, but I promise I'll put something there soon. :sweat_smile: